pkg/tenant/api/v1/api.go

```diff
@@ -48,18 +45,19 @@ type StatusReply struct {
 // Tenant Requests and Responses
 //===========================================================================
 
-type User struct {
-	ID       string `json:"id" uri:"id" binding:"required"`
-	UserName string `json:"user_name"`
+type Tenant struct {
+	ID              string `json:"id" uri:"id" binding:"required"`
+	TenantName      string `json:"tenant_name"`
+	EnvironmentType string `json:"environment_type"`
 }
 
-type UserQuery struct {
-	Query         string
-	NextPageToken string
+type PageQuery struct {
+	PageSize      uint32 `url:"page_size"`
+	NextPageToken string `url:"next_page_token"`
 }
```

Comment:
```suggestion
type PageQuery struct {
	PageSize      uint32 `url:"page_size,omitempty"`
	NextPageToken string `url:"next_page_token,omitempty"`
}
```

I'm not sure, but I think that adding `omitempty` will ensure that the query is not added to the URL if the `PageSize` is 0 or `NextPageToken` is empty. This will create better URLs that are easier to parse in the backend (if it's not `omitempty` then it's probably a different struct tag). Note that will probably affect the test you wrote; you will probably have to pass a non-zero/non-nil page-query to fix the test.